lib/hs/thrift.cabal

```diff
@@ -18,8 +18,8 @@
 --
 
 Name:           thrift
-Version:        1.0.0-dev
-Cabal-Version:  >= 1.24
+Version:        0.99.0
```

Comment:
@jfarrell @Jens-G note: cabal will no longer allow "1.0.0-dev" as a version... this happens with newer versions of cabal.